media-gfx/blender/blender-9999.ebuild

```diff
@@ -4,8 +4,11 @@
 EAPI=8
 
 PYTHON_COMPAT=( python3_{11..12} )
+# matches media-libs/osl
+LLVM_COMPAT=( {15..17} )
+VIRTUALX_REQUIRED=manual
```

Comment:
Is this variable needed now that you removed the explicit calls to `virtx`?

Reply:
Fixed.